doc/dev/data-structures.md

```diff
@@ -362,6 +362,37 @@ synchronised.
 
 Continue reading [with the error handling](error-handling.md).
 
-## Order Preserving Minimal Perfect Hash Map
+## Order Preserving Minimal Perfect Hash Map (aka OPMPHM)
 
 All structs are defined in [opmphm.h](/src/include/kdbopmphm.h).
+
+Generally the OPMPHM is a arbitrary function.
```

Comment:
Do you mean "OPMPHM is a general function"? The sentence currently sounds like it would be okay if OPMPHM calculates Fibonacci numbers.